meson.build

```diff
@@ -0,0 +1,14 @@
+project('gtk-layer-shell', ['c'], version: '0.1.0', license: 'MIT',
+	default_options: ['c_std=c99', 'warning_level=2'])
+
+gtk = dependency('gtk+-wayland-3.0', version: '>=3.22.0')
+wayland_client = dependency('wayland-client', version: '>=1.10.0')
+wayland_scanner = dependency('wayland-scanner', version: '>=1.10.0', required: false)
+
+pkg_config = import('pkgconfig')
+gnome = import('gnome')
```

Comment:
I'm guessing this is just to generate the GIR? If so, can you add a comment indicating this? Also, would it make sense to make this somehow optional in case `gnome` isn't available?

Reply:
It's an inherent part of meson, it can't be unavailable

Reply:
Cool cool